libp2p/topicsHandler.go

```diff
@@ -0,0 +1,118 @@
+package libp2p
+
+import (
+	"sync"
+)
+
+// TODO[Sorin]: add unit tests
+type topicsHandler struct {
```

Comment:
I would not use this component as to use old critical sections
Or, I would make it unexported and let only the map definitions without the mutex usage

Reply:
dropped topicsHandler